atspi-common/src/events/object.rs

```diff
@@ -107,9 +107,41 @@ pub enum Property {
 	Other((String, OwnedValue)),
 }
 
+impl Clone for Property {
+	fn clone(&self) -> Self {
+		match self {
+			Property::Name(name) => Self::Name(name.clone()),
+			Property::Description(description) => Self::Description(description.clone()),
+			Property::Role(role) => Self::Role(*role),
+			Property::Parent(parent) => Self::Parent(parent.clone()),
+			Property::TableCaption(table_caption) => Self::TableCaption(table_caption.clone()),
+			Property::TableColumnDescription(table_column_description) => {
+				Self::TableColumnDescription(table_column_description.clone())
+			}
+			Property::TableColumnHeader(table_column_header) => {
+				Self::TableColumnHeader(table_column_header.clone())
+			}
+			Property::TableRowDescription(table_row_description) => {
+				Self::TableRowDescription(table_row_description.clone())
+			}
+			Property::TableRowHeader(table_row_header) => {
+				Self::TableRowHeader(table_row_header.clone())
+			}
+			Property::TableSummary(table_summary) => Self::TableSummary(table_summary.clone()),
+			Property::Other((property, value)) => Self::Other((
+				property.clone(),
+				value.try_clone().unwrap_or_else(|err| {
+					eprintln!("Failed to clone value: {err:?}");
+					u8::default().into()
```

Comment:
Again, I would argue that we should give the `.try_clone()` option to users and let them handle errors.

On the other hand, it's not a trait (in `std` anyways), and it is exceedingly rare that this would actually be triggered.... Hmmmm

Does cloning a file descriptor *do* anything? Wouldn't the "limit" only be exceeded if we tried to open it?

Reply:
AFAIU, having the `OwnedFd` is `having an open file`, otherwise cloning these would not be limited?

I fully agree that the chances are slim. Have we ever seen file descriptors being sent? 
If we can, I would like to not burden our users with `try_clone()` but just keep a `Clone` implementation.